packages/@sanity/types/src/schema/definition/type/block.ts

```diff
@@ -18,10 +18,10 @@ export interface BlockRule extends RuleDef<BlockRule, any[]> {}
 export interface DecoratorDefinition {
   title: string
   value: string
-  icon?: ReactNode | ComponentType
-  portableText?: {
-    icon?: ReactNode | ComponentType
-    render?: ComponentType
+  icon?: ReactNode | ComponentType<any>
+  blockEditor?: {
```

Comment:
Is this correct, @skogsmaskin ? I guess these are about to change real soon?

Reply:
They are! It's OK to merge it like this for now though.

Reply:
Yeah, changed these to what works for runtime in the current version. Using portableText crashes the studio with an unhelpful error message.